shoes-core/spec/shoes/animation_spec.rb

```diff
@@ -60,7 +60,7 @@
   it { is_expected.not_to be_stopped }
 
   describe "with framerate" do
-    let(:opts) { {:framerate => 36, :app => app} }
+    let(:opts) { { :framerate => 36, :app => app } }
```

Comment:
Don't recall a specific decision, but I can go either way on it so long as it's consistent between begin and end, which this does.